pkgs/applications/window-managers/hyprwm/hyprland-plugins/hyprfocus.nix

```
@@ -17,6 +18,13 @@ mkHyprlandPlugin hyprland {
     hash = "sha256-qIl7opF7fA1ZmC91TGQ7D12tB7kHc6Sn9DrfUN6sbBY=";
   };
 
+  patches = [
+    (fetchpatch2 {
+      url = "https://github.com/pyt0xic/hyprfocus/pull/15.diff";
```

```suggestion
      url = "https://github.com/pyt0xic/hyprfocus/commit/e7d9ee3c470b194fe16179ff2f16fc4233e928ef.patch";
```
If the PR changes then `15.diff` will change. Use the commit instead

---

Oh, you're right! Done